processing/src/test/java/org/apache/druid/segment/nested/NestedFieldLiteralColumnIndexSupplierTest.java

```diff
@@ -726,6 +808,26 @@ public void testSingleValueLongWithNullRangeIndex() throws IOException
     ImmutableBitmap bitmap = forRange.computeBitmapResult(bitmapResultFactory);
     checkBitmap(bitmap, 0, 6, 7);
 
+    forRange = rangeIndex.forRange(100, true, 300, true);
+    Assert.assertEquals(0.0, forRange.estimateSelectivity(10), 0.0);
+    bitmap = forRange.computeBitmapResult(bitmapResultFactory);
+    checkBitmap(bitmap);
+
+    forRange = rangeIndex.forRange(100, false, 300, true);
+    Assert.assertEquals(0.2, forRange.estimateSelectivity(10), 0.0);
+    bitmap = forRange.computeBitmapResult(bitmapResultFactory);
+    checkBitmap(bitmap, 0, 6);
```

Comment:
I see this expansion of the ranges checked for the single typed columns.  When I looked at the broader file, I didn't see a similar test for the variant typed varieties.  Is there a reason it doesn't make sense to have a similar style of range test checker thingie for a variant column too?  Maybe have a pure 3-typed variant, a String+Long, String+Double and Long+Double?

Reply:
Right... the range-style filters fall back to those other interfaces.